### PR TITLE
Remove paraview

### PR DESCRIPTION
**Pull request by Pasarus**
Does was it says on the tin.

### CODE REVIEW

File: development/readme.md [hunk 1]
```diff
@@ -9,11 +9,8 @@ There are images for each base OS:
 - [`mantidproject/mantid-development-centos7`](https://hub.docker.com/r/mantidproject/mantid-development-centos7/) - CentOS 7
 - [`mantidproject/mantid-development-ubuntubionic`](https://hub.docker.com/r/mantidproject/mantid-development-ubuntubionic/) - Ubuntu 18.04 (Bionic)
 
-Typically you'd want to use the `latest` tag.
```
Comment (martyngigg) [inline development/readme.md:12]: Can we keep the sentence about "Typically you'd want to use the `latest` tag."?

Comment (martyngigg) [reply development/readme.md:12]: Actually the next one too, just removing the bit about paraview from the sentence.

File: development/readme.md [hunk 2]
```diff
@@ -58,7 +55,7 @@ If you don't want to or can't use `x11docker` then you can try using simple X se
 ### Python 3
```
Comment (martyngigg) [inline development/readme.md:55]: It's not technically related but the Python 3 stuff is not required anymore so lets just remove that section?

Comment (Pasarus) [reply development/readme.md]: Done!

File: development/readme.md (hunk 1 above)
Comment (Pasarus) [reply development/readme.md:12]: Done!

**Review by martyngigg — APPROVED** [review]

**Pull request merged**